docs/releases/1.9.txt

```diff
@@ -260,6 +260,9 @@ Miscellaneous
   :class:`~django.forms.SelectDateWidget` in ``django.forms.widgets``
   (or simply ``django.forms``) instead.
 
+* ``django.db.models.fields.add_lazy_relation`` is deprecated. The new function
+  ``lazy_related_operation`` should be used instead.
```

Comment:
Since it's a private API, I don't think we should mention it in the documentation. The warning message suffices.